cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -959,7 +967,7 @@ nodeInfoLabels =
            , padTop (T.Pad 1) $ txt "epoch / slot:"
            ,                    txt "block number:"
            ,                    txt "chain density:"
-           , padTop (T.Pad 1) $ txt "blocks minted:"
+           , padTop (T.Pad 1) $ txt "blocks minted / inv.:"
```

Comment:
Particularly in the UI, we should be careful with labels and not say they're invalid.

Reply:
Agree. I'll pick another label.

Reply:
Done.